src/App.js

```diff
@@ -1,25 +1,41 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+/* eslint-disable comma-dangle */
+import React from "react";
+
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from "react-router-dom";
+import RootLayout from "./Layout/RootLayout";
+import Appointment from "./components/appointment/Appointment";
+import MainPage from "./components/pages/MainPage";
+import SingleDoctorPage from "./components/pages/SingleDoctorPage";
+import AddDoctorForm from "./components/doctor/form";
+
+import MyAppointment from "./components/appointment/MyAppointment";
+
+import DoctorsList from "./components/doctor/doctor_list";
+import Login from "./components/login/login";
+import Register from "./components/login/register";
+
+export const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<RootLayout />}>
+      <Route path="/my-appointments" element={<MyAppointment />} />
+      <Route path="/delete-doctor" element={<DoctorsList />} />
+      <Route index element={<MainPage />} />
+      <Route path="/add-doctor" element={<AddDoctorForm />} />
+      <Route path="/doctors/:id" element={<SingleDoctorPage />} />
+      <Route path="/appointment" element={<Appointment />} />
+      <Route path="/login" element={<Login />} />
+      <Route path="/register" element={<Register />} />
+    </Route>
+  )
+);
+
+function App() {

```

Comment:
- [ ]  Please, kindly refactor this main "App" component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the other functions in your project to ES6.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 11 total): 

 ![image](https://github.com/sumon766/frontend-doctor-appointment/assets/57421823/ae90b2fd-e694-401a-bfab-6faec85ac58f)


That way, your code will meet the latest standards. 💪